### PR TITLE
[COLLECTIONS-749] better documentation for signedness

### PR DESCRIPTION
**Pull request by Claudenw**
Updated documentation for COLLECTIONS-749.

fixes COLLECTIONS-749.

### CODE REVIEW

File: src/main/java/org/apache/commons/collections4/bloomfilter/hasher/HashFunctionIdentity.java [hunk 1]
```diff
@@ -58,6 +58,10 @@
 
     /**
      * Identifies the signedness of the calculations for this function.
+     * <p>
+     * When the hash function executes it generally returns an array of bytes.
+     * That array is converted into one or more numerical values.
+     * The Signedness identifies if those values are signed or unsigned.
```
Comment (garydgregory) [inline src/main/java/org/apache/commons/collections4/bloomfilter/hasher/HashFunctionIdentity.java]: Close HTML tags.

**Comment by aherbert** [comment]
I think this PR is obsolete. Master now contains very similar text to that in this PR.

**Comment by kinow** [comment]
ping @Claudenw, see @aherbert 's comment above :point_up:

**Comment by garydgregory** [comment]
This remaining change must have already been done by a different commit as it is in git master.

**Pull request closed without merging**